### PR TITLE
Added gamedev-wg

### PR DESCRIPTION
**Pull request by AlexEne**
I just copied the format from the other working groups. I am not sure how to test that everything is fine.
I assumed the usernames are the github usernames.

That address doesn't exist yet so I think we shouldn't merge this until it exists.
`gamedev-wg@rust-lang.org`

Thank you,
Alexandru Ene

### CODE REVIEW

**Comment by Manishearth** [comment]
I'm confused: there's no charter or anything for this group yet? You also would need to see if other people in the community are interested in being involved.

https://github.com/rust-lang/wg-governance/issues/10

**Comment by AlexEne** [comment]
It is related to the rust-lang/wg-governance#10 issue. We just had an initial meeting with @skade about this and he suggested setting this up here as one of the first steps.

File: teams/wg-gamedev.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name = "wg-gamedev"
+wg = true
+
+[people]
+leads = ["alexene"]
```
Comment (pietroalbini) [inline teams/wg-gamedev.toml]: Please capitalize your GitHub username (as it's shown in the UI).

File: teams/wg-gamedev.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+members = [
+    "erlend-sh"
```
Comment (pietroalbini) [inline teams/wg-gamedev.toml]: You also need to be listed as a member.

**Comment by Manishearth** [comment]
Ah, I see. So to be clear, this isn't to be merged yet.

**Comment by AlexEne** [comment]
Yes, We can keep from merging this until a charter is available if merging into this repo requires a charter to be available. 

To be honest this is the one of the first WG to be done this way so I am unsure what comes first (charter vs pr here vs github org, etc.). I am trying to get done as much as possible from the organizational side out of the way so we can move forward so that's why I did this as one of the first PRs :D

**Comment by Manishearth** [comment]
You'll need a charter first, and also probably more members (you'll probably gain members when the charter is posted for others to see)

**Comment by AlexEne** [comment]
Alright, I'll fix up this PR and we can come back on this after the meeting next week where we will discuss and move towards finalizing the charter.

File: teams/wg-gamedev.toml (hunk 1 above)
Comment (AlexEne) [reply teams/wg-gamedev.toml]: Ok, thanks, I thought it's the alexene (as in the .toml) file.

Comment (pietroalbini) [reply teams/wg-gamedev.toml]: The file name also needs to be changed. This should have been caught by `check` but username validation is not implemented yet.

Comment (AlexEne) [reply teams/wg-gamedev.toml]: Yup, I managed to eventually do it, thanks.

**Comment by skade** [comment]
Ha, that was quick. Sorry for the confusion. We need to figure out how to represent the WG as "in formation" first.

**Comment by pietroalbini** [comment]
Note that now you need to add `github-id` to the added people.

**Comment by pietroalbini** [comment]
Hi! Sorry for the trouble, but we made changes to this repository's CI, and the PR needs to be rebased on top of master. Could you do that? Thanks!

**Comment by AlexEne** [comment]
Yes, sorry about the lack of activity on this one as we were focused on some other things. I will try to have something ready this weekend

**Comment by AlexEne** [comment]
I re-opened this https://github.com/rust-lang/team/pull/124 here, as it was too painful to deal with what went wrong on this PR.

**Pull request closed without merging**